lib/ecto/migration.ex

```diff
@@ -1094,6 +1094,9 @@ defmodule Ecto.Migration do
       `:nothing` (default), `:delete_all`, `:nilify_all`, or `:restrict`.
     * `:on_update` - What to do if the referenced entry is updated. May be
       `:nothing` (default), `:update_all`, `:nilify_all`, or `:restrict`.
+    * `:validate` - Whether or not to validate the foreign key constraint on
+       creation or not. Only available in postgres, and should be followed by
```

Comment:
```suggestion
       creation or not. Only available in PostgreSQL, and should be followed by
```

Reply:
Looks good. We should probably raise if validate is set for MySQL and TDS. Can you please update them accordingly?

Reply:
Updated. I'm not sure the approach was the best, but I also don't quite know where else to put it.